File: ScpProject/DAL/DTOs/Messages/GroupMessagePreviewDTO.cs
using System;

namespace DAL.DTOs.Messages
{
    public class GroupMessagePreviewDTO
    {
        public String MessageGroupTitle { get; set; }
        public int MessageGroupId { get; set; }
        public int LastSentByUserId { get; set; }
        public int MessageId { get; set; }
        public string LastSentFirstName { get; set; }
        public string LastSentLastName { get; set; }
        public int UserId { get; set; } //used to say if itwas from an athlete or a coach
        public int AthleteId { get; set; }  //used to say if itwas from an athlete or a coach
        public string MessageContent { get; set; }
        public DateTime SentTime { get; set; }
        public int ViewerId { get; set; }
        public bool ReadOnly { get; set; }
        public bool Pause { get; set; }
        public DateTime? ReadTime { get; set; }
        public Guid SignalRGroupId { get; set; }
    }
}

File: ScpProject/StrengthConApp/src/app/Models/Messages/GroupMessagePreviewDTO.ts
export class GroupMessagePreviewDTO {
    public MessageGroupTitle: string = '';
    public LastSentFirstName: string = '';
    public LastSentLastName: string = '';
    public MessageContent: string = '';
    public MessageGroupId: number = 0;
    public LastSentByUserId: number = 0;
    public MessageId: number = 0;
    public UserId: number = 0;
    public AthleteId: number = 0;
    public SentTime: Date;
    public Selected: false;
    public ViewerId: number = 0;
    public ReadOnly: boolean = false;
    public Pause: boolean = false;
    public SignalRGroupId : string = '';
}